src/test/java/core/basesyntax/MyCalculatorTest.java

```diff
@@ -0,0 +1,1275 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import org.junit.jupiter.api.Test;
+
+public class MyCalculatorTest {
+    private static final MyCalculator CALCULATOR = new MyCalculator();
```

Comment:
```suggestion
    private static final MyCalculator calculator = new MyCalculator();
```
не immutable класи ловеркейсом пишуться